src/goto-programs/convert_nondet.cpp

```diff
@@ -132,26 +133,23 @@ static void convert_nondet(
   }
 }
 
-
-
 void convert_nondet(
-  goto_functionst &goto_functions,
-  symbol_tablet &symbol_table,
+  goto_modelt &goto_model,
   message_handlert &message_handler,
   size_t max_nondet_array_length,
   size_t max_nondet_tree_depth)
 {
-  for(auto &goto_program : goto_functions.function_map)
+  for(auto &goto_program : goto_model.goto_functions.function_map)
   {
     convert_nondet(
       goto_program.second.body,
-      symbol_table,
+      goto_model.symbol_table,
       message_handler,
       max_nondet_array_length,
       max_nondet_tree_depth);
   }
 
-  goto_functions.compute_location_numbers();
+  goto_model.goto_functions.compute_location_numbers();
```

Comment:
We have `remove_skip` and `remove_unused_functions`, which both forward data members to functions of the same name. In the interests of consistency (and closer adherence to the Law of Demeter), I wonder whether we should use a similar approach here (and on line 142), by adding `compute_location_numbers` and `get_function_map` as direct members of goto_model, which can then forward to the appropriate method/member on `goto_functions`.